src/main/python/run_trecqa.py

```diff
@@ -293,6 +292,6 @@ def get_stopwords():
                 except KeyError as e:
                     print("Pattern not found for question: {}".format(e))
 
-        eval_depth = [int(i) for i in args.k.strip().split()]
+        eval_depth = map(int, args.k.strip().split())
```

Comment:
In Python 3, map returns a generator and not a list, make sure this is what you want.